CHANGELOG.next.md

```diff
@@ -21,6 +21,8 @@ Thanks, you're awesome :-) -->
 
 #### Improvements
 
+* Extended `file.pe`. #1071
+
```

Comment:
We can remove that. During the RFC process there's no official changes to advertize